proto/buf.lock

```diff
@@ -9,5 +9,5 @@ deps:
   - remote: buf.build
     owner: grpc-ecosystem
     repository: grpc-gateway
-    commit: 7dec1bb6f79840c5849dac18f642de9c
-    digest: shake256:aa37e1bbccaf5793f1c82af9e620e67220f390dc5cbab2b8b3091aa4c030f32d9144be4dbfb8a90d02d44cd6b781875959ab1d454742ebd3d586b1a7806a07ea
+    commit: ca289928665845a7b2f7285d052c739a
+    digest: shake256:67b115260e12cb2d6c5d5ce8dbbf3a095c86f0e52b84f9dbd16dec9433b218f8694bc9aadb1d45eb6fd52f5a7029977d460e2d58afb3208ab6c680e7b21c80e4
```

Comment:
this file was added by precommit automatically after my first commit. not sure why this happened.